deployment/.golangci.yml

```diff
@@ -26,176 +25,188 @@ linters:
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Debugw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Infow
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Warnw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Errorw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Panicw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.Logger).Fatalw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.SugaredLogger).AssumptionViolationw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.SugaredLogger).Tracew
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.SugaredLogger).Criticalw
-      - (github.com/smartcontractkit/chainlink-common/pkg/logger.SugaredLogger).With
-  nolintlint:
-    require-specific: true
-    require-explanation: true
-issues:
-  exclude-rules:
-    # when performing simple refactor, the usage of addressbook will fail CI, this is a temporary ignore
-    - text: "SA1019: (.*).ExistingAddresses is deprecated: AddressBook is deprecated"
-      linters:
+      - linters:
           - staticcheck
-    - text: "^var-naming: don't use an underscore in package name"
-      linters:
+        text: 'SA1019: (.*).ExistingAddresses is deprecated: AddressBook is deprecated'
+      - linters:
+          - revive
+        text: '^var-naming: don''t use an underscore in package name'
+      - linters:
+          - revive
+        path: memory/(.+)\.go
+      - linters:
+          - gosec
+        text: '^G404: Use of weak random number generator'
+      - linters:
+          - govet
+        text: declaration of "err" shadows
+      - linters:
           - revive
-    - path: memory/(.+)\.go
-      linters:
-        - revive
-    - text: "^G404: Use of weak random number generator"
-      linters:
-        - gosec
-    - linters:
-        - govet
-      text: "declaration of \"err\" shadows"
-    # ignore false positive by revive on the function names (ID, Version, Description) clash between sequence and operation
-    - path: operations/operation.go
-      text: "^confusing-naming:"
-      linters:
-        - revive
-    - path: operations/sequence.go
-      text: "^confusing-naming:"
-      linters:
-        - revive
-    # ignore false positive by revive on function names for BaseDataStore interface implementations
-    - path: datastore/memory_datastore.go
-      text: "^confusing-naming:"
-      linters:
-        - revive
+        path: operations/operation.go
+        text: '^confusing-naming:'
+      - linters:
+          - revive
+        path: operations/sequence.go
+        text: '^confusing-naming:'
+      - linters:
+          - revive
+        path: datastore/memory_datastore.go
+        text: '^confusing-naming:'
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
```

Comment:
Where are these coming from? Did the tool add them because v1 did it by default or something?
I think we can drop them safely :shrug: